homeassistant/components/intellifire/manifest.json

```diff
@@ -11,5 +11,5 @@
   "documentation": "https://www.home-assistant.io/integrations/intellifire",
   "iot_class": "local_polling",
```

Comment:
Can I have multiple iot_classes?

When its in local mode its `local_polling` but obviously if I use cloud control we get `cloud_polling` ... not sure what to do here.